content/operate/rc/security/access-control/saml-sso/saml-integration-okta-generic.md

```diff
@@ -272,14 +269,8 @@ To activate SAML, you must have a local user (or social sign-on user) with the *
 
     {{<image filename="images/rc/saml/okta_saml_app_int_15.png" >}}
 
-1. If only one user is defined in Redis Cloud console, you get a popup window where you select **Confirm** to convert the local user to a SAML user. 
-
-    > Consider setting up one more local user other than a SAML user.
+1. If only one user is defined in Redis Cloud console, you get a popup window where you select **Confirm** to convert the local user to a SAML user. Consider setting up one more local user other than a SAML user.
```

Comment:
Bit awkward again? Maybe something like "If only one user is defined in Redis Cloud console, you will also see a popup window asking if you want to convert the local user to a SAML user. Select **Confirm** to go ahead with the conversion."